oak/server/wasm_node_fuzzer.cc

```diff
@@ -5,6 +5,6 @@
 // See https://llvm.org/docs/LibFuzzer.html#fuzz-target.
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   std::string module(reinterpret_cast<char const*>(data), size);
-  std::unique_ptr<oak::WasmNode> node = oak::WasmNode::Create("test", module);
+  std::unique_ptr<oak::WasmNode> node = oak::WasmNode::Create(nullptr, "test", module);
```

Comment:
Can you add a comment why it's fine to pass `nullptr` here? Presumably this is not going to be enough at some point when the wasm node needs to call some methods on it? Or is it always going to be used purely for ownership?

Reply:
Added a comment.

Not having an owning-runtime reference will only be relevant if the node runs and wants to invoke (say) the `node_create` host function.  I don't believe that's going to happen here (in the fuzz target).

Reply:
Why not? When we offer a wasm API to create nodes (which you are already adding), then it may well end up being invoked as part of the fuzzing. At that point I guess we may need to pass in a mock or some other fake object with the appropriate methods?

Reply:
Well, if you're planning to extend the fuzz target so it does more than just loading Wasm code then we might need to re-visit, but I didn't know there were any plans to do that (particularly as doing more would be quite a big deal &ndash; heading towards something syzkaller-like would probably be needed to get decent coverage).